### DISCUSSION TITLE
Call to a member function getRelationModel () on bool

### DISCUSSION THREAD

**Discussion by AlexisZverev** [YetiForceCompany/YetiForceCRM#15509] [General]
On the Yetiforce 6.2
After creating a new project in the Project Manager section, it shows an error: Call to a member function getRelationModel () on bool

Has anyone encountered a similar problem?
Perhaps there is a solution?
With what it can be connected?
I'm a beginner, and I don't even know where to dig yet
![Image 3](https://user-images.githubusercontent.com/74072835/144448828-c9800846-b649-4cfc-ae28-c05abd95f047.jpg)

**Comment by mariuszkrzaczkowski** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/discussions/15377
